lib/Parser/Parse.cpp

```diff
@@ -3659,9 +3659,9 @@ ParseNodePtr Parser::ParseArgList( bool *pCallOfConstants, uint16 *pSpreadArgCou
         CHAKRATEL_LANGSTATS_INC_LANGFEATURECOUNT(SpreadFeatureCount, m_scriptContext);
     }
 
+    *pCount = (uint16)count;
```

Comment:
I consider it optional since you just moved it but maybe we want to do a C++ style cast here